securite/l-equipe-a-entame-une-demarche-sur-monservicesecurise.md

```diff
@@ -1 +1,12 @@
-# L'équipe a entamé une démarche sur MonServiceSécurisé.
\ No newline at end of file
+# L'équipe a entamé une démarche sur MonServiceSécurisé.
+
+## Description
+
+Le produit est inscrit sur [MonServiceSécurisé](https://monservicesecurise.cyber.gouv.fr/)
```

Comment:
est-ce qu'on reprend ça ? https://doc.incubateur.net/communaute/gerer-son-produit/les-standards/securite/securite-and-homologation#comment-securiser-et-homologuer-votre-service-avec-monservicesecurise